asf_core_data/pipeline/preprocessing/feature_engineering.py

```diff
@@ -353,9 +356,11 @@ def get_heating_features(df, fine_grained_HP_types=False):
 def get_postcode_coordinates(df):
     """Add coordinates (longitude and latitude) to the dataframe
     based on the postcode.
+    Postcode fields must be named POSTCODE.
 
     Args:
         df (pandas.DataFrame): EPC dataframe.
```

Comment:
Maybe calling it `epc_data` or `epc_df`?

Reply:
Doesn't necessarily have to be EPC data, could be MCS. But I'll adapt so you can pick the POSTCODE variable name.